cmd/zar/README.md

```diff
@@ -261,15 +265,15 @@ zar ls custom.zng
 ```
 To see what's in it:
 ```
-zq -f table $ZAR_ROOT/20180324/1521911772.980384.zng.zar/custom.zng | head -10
+find $ZAR_ROOT -name 'custom.zng' | head -n 1 | xargs zq -f table - | head -n 10
 ```
 You can see the IPs, counts, and _path strings.
 
 At the bottom you'll also find a record describing the micro-index layout. To
 see it:
 
 ```
-zq -f table $ZAR_ROOT/20180324/1521911772.980384.zng.zar/custom.zng | tail -2
+find $ZAR_ROOT -name 'custom.zng' | head -n 1 | xargs zq -f table - | tail -n 2
```

Comment:
```suggestion
find $ZAR_ROOT -name custom.zng | head -n 1 | xargs zq -f table 'tail 2' -
```

Reply:
@nwt: I'm fine with moving the `tail` into the ZQL, but I'll be making this `tail 1` instead of the `tail 2` shown in the suggestion, since the goal was just to show that final record.